_docs/Threat-Model.md

```diff
@@ -1,70 +1,126 @@
 ---
 title: Threat Model
 ---
-## Scope of the Threat Model
+#Threat Model
+This model shall provide an overview on the system architecture regarding security. It shall list the attackers and threats Qabel opposes and their capabilities as well as security assumptions. It also shall outline the targeted security goals by describing how they are reached.
 
-The threat model exhaustively lists what an attacker can learn about a user.
-As the computers running a Qabel client or server could be compromised by an
-attacker, the threat model also lists the capabilities of Qabel clients and
-servers.
+##Security Goals
+The main targeted goal of Qabel is to require no trust in any entity in order to achieve **confidentiality**, **integrity**, **authenticity** and partly **anonymity**. The first three goals are reached by using Authenticated end-to-end Encryption. The fourth goal cannot fully be reached due to for example storage write access restriction to the authenticated user who pays for the storage. Thus quota is tracked per user.
 
-This document does not list other things an attacker could achieve, like
-blocking access to a Qabel server. It also assumes that the user doesn't do
-something as stupid as uploading the secret key material onto a public server.
+###Restrictions to Freedom of Trust
+The mentioned main goal is limited by unavoidable restrictions like the trust in proper storage of data on the server and possible malware on the client. Hence following trust relationships are required to achieve the mentioned security goals:
+* Trust in the client device: the newest original Qabel client software is installed, no malware is installed.
+* Trust in the Qabel servers: all sent data is stored properly i.e.: drop messages are stored for sufficient duration on the specified drop server, files are stored permanently (until they are deleted by the client) in the specified directory.
+* Trivial assumptions: users keep their credentials and keys secret.
 
-> Note: The existing Qabel software doesn't implement these requirements yet,
-> and this document is still in a state of flux.
-> Please add and question existing entries.
+####Further Assumptions
+TLS certificates of Qabel servers are not forged by trusted CAs.
```

Comment:
It should be explained why we need this. What does it add, when Qabel already provides  end-to-end authenticated encryption.